tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_jump_opcodes.py

```diff
@@ -0,0 +1,166 @@
+"""
+EOF Jumpcode tests
+"""
+
+import pytest
+
+from ethereum_test_tools import EOFException, EOFTestFiller
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools.eof.v1 import Container, Section
+
+from .. import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-3540.md"
+REFERENCE_SPEC_VERSION = "8dcb0a8c1c0102c87224308028632cc986a61183"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+@pytest.mark.parametrize(
+    "eof_code,expected_hex_bytecode,exception",
+    [
+        pytest.param(
```

Comment:
Already covered by:
tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjump.py::test_rjump_positive_negative

Reply:
there negative value comes after stop, here it is before